reana_commons/k8s/volumes.py

```diff
@@ -85,9 +86,13 @@ def get_workspace_volume(workflow_workspace):
     :returns: Tuple consisting of the Kubernetes volumeMount and the volume.
     """
     volume_mount = {"name": "reana-workspace-volume", "mountPath": workflow_workspace}
-
+    mounted_paths = extract_mounted_paths()
+    host_workspace_path = workflow_workspace
+    for host_path, mounted_path in mounted_paths.items():
+        if host_workspace_path.startswith(mounted_path):
```

Comment:
I added this line because in case that `/tmp/reana:/tmp/reana/eos` and `/var/eos:/eos` has been mounted, and I would like to run a workspace with the `/tmp/reana/eos` the `/eos` part would be also replaced. So it only replaces in case it starts with the wanted path